resources/views/community/components/message/list-page.blade.php

```diff
@@ -0,0 +1,123 @@
+@props([
+    'messages' => [],
+    'currentPage' => 1,
+    'totalPages' => 1,
+    'unreadCount' => 0,
+    'totalMessages' => 0,
+])
+
+<?php
+
+use App\Site\Enums\UserPreference;
+use App\Site\Models\User;
+use Illuminate\Support\Carbon;
+
+$user = request()->user();
+$isShowAbsoluteDatesPreferenceSet = BitSet($user->websitePrefs, UserPreference::Forum_ShowAbsoluteDates);
+$monthAgo = Carbon::now()->subMonth(1);
+
+?>
+
+<script>
+function deleteMessage(id) {
+    if (confirm('Are you sure you want to delete this message?')) {
+        $.post('/request/message/delete.php', {
+            thread_id: id,
+            user: "{{ $user->User }}"
+        })
+    }
+}
+</script>
+
+<x-app-layout
+    pageTitle="Messages"
+    pageDescription="Messages involving {{ $user->User }}"
+>
+    <x-user.breadcrumbs targetUsername="{{ $user->User }}" currentPage="Messages" />
+
+    <div class="mt-3 w-full flex gap-x-3">
+        {!! userAvatar($user->User, label: false, iconSize: 48, iconClass: 'rounded-sm') !!}
+        <h1 class="mt-[10px] w-full">{{ $user->User }}'s Messages</h1>
```

Comment:
Yes. It should only be visible to the current user. I felt this was consistent with the other pages under the User breadcrumb, although Messages could be its own top-level breadcrumb item instead of being nested under the User.

Reply:
Not sure if it sways any opinions, but here's an illustration showing the mapping from the breadcrumb. "You" does appear in the subheading to clarify context.
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/db365d9c-05a2-41f2-9b53-cfe7814d368d)

Reply:
My opinion is unswayed 😅 But this isn't a dealbreaker by any means 🙂

When I see "WCopeland's Messages", my inner voice is saying "of course it is, I'm the only one who can see the page!" Which leads me to the question... am I _really_ the only one who can see the page? Why else would the page remind me in the `<h1>` of who I am?

It is for this reason that on the completion progress page, if you're viewing your own page, the label changes:

![Screenshot 2023-11-29 at 4 58 19 PM](https://github.com/RetroAchievements/RAWeb/assets/3984985/fcfd4aef-053d-43b9-92d1-3053a4a6ba2d)

Obviously, I'm heavily speaking from my biases in this conversation, but I don't perceive this as confusing. I think it's a better experience and appreciate the attention to detail when the app recognizes my context. This doesn't necessarily mean we have to have "Your X" or "My X" labels everywhere to @luchaos's point. But in this case, this is exclusively context-driven content which I believe is what makes it appropriate.

Having the `<h1>` just read "Messages" feels like a good compromise.

Reply:
> But in this case, this is exclusively context-driven content which I believe is what makes it appropriate.

That's fair. In those two examples in the screenshots i would've opted to be much more stern for brevity and less noise. I can see its merit though, too.
I don't have a strong opinion on that at this point.

Reply:
The i18n angle is a fair point and not something I had previously considered.

Reply:
Changed to "Messages" and removed the icon/breadcrumb.